test/upload_validation_test.ts

```diff
@@ -806,6 +843,10 @@ describe('Pack metadata Validation', () => {
               'alphanumeric characters, underscores, and dashes, and no spaces.',
             path: 'syncTables[0].schema.identity.name',
           },
+          {
```

Comment:
I cannot figure out why this test is now also failing the schema parsing around the getter's item schema of all things, especially because correcting the identity name (an unrelated field) results in no errors occurring.

Reply:
I can take a look.

Reply:
Aha, it's because makeSyncTable() copies the identity into the schema, which gets copied into the getter's item schema. So the item schema is actually failing validation for the same reason--it contains an identity whose name has spaces. Our zod error transforming is properly surfacing in the invalid identity name error at the top level, but it's not doing it for child fields. I think this is ok, maybe long term something to try to fix, but the items schema really does have an error so I think it's ok that we're flagging an error on it in this test.